101-hdinsight-linux-with-edge-node/azuredeploy.json

```diff
@@ -0,0 +1,227 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "clusterName": {
```

Comment:
What error are you getting?  Other templates seem to be making it through CI generating param values...